hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/init/Contexts.java

```diff
@@ -225,6 +231,16 @@ private static String tempDirPrefix(String jetInstanceName, String jobId, String
                     + "-" + jobId
                     + "-" + resourceId.substring(0, min(32, resourceId.length())).replaceAll("[^\\w.\\-$]", "_");
         }
+
+        @Nonnull @Override
+        public ManagedContext managedContext() {
+            return serializationService.getManagedContext();
```

Comment:
It would be safer and easier to get the managed context from the `jetInstance()`. Theoretically the serializationService could have different managedContext. It's not the member's serialization service. IMO, the managedContext shouldn't be published by it. Maybe we can even use `SerializationService` instead of `InternalSerializationService`, do we use any other method than `toObject` and `toData`?